lib/cdmdexer/default_solr.rb

```diff
@@ -4,7 +4,7 @@ module CDMDEXER
   # Commnicate with Solr: add / delete stuff
   class DefaultSolr
     attr_reader :url, :client
-    def initialize(url: 'http://localhost:8983/solr/core-here', client: RSolr)
+    def initialize(url: 'http://localhost:8983/solr/blacklight-core', client: RSolr)
```

Comment:
Instead of defaulting to `blacklight-core`, could we use something like `ENV['SOLR_URL'] || 'http://localhost:8983/solr/core` to allow this to use the environment if available? I'm not even sure that would work in this context but using `blacklight-core` here doesn't sit totally right with me since this isn't supposed to be a gem for BL only.

Reply:
Yeah, I'll iron that out here.

Reply:
Resolved in [549a973](https://github.com/UMNLibraries/cdmdexer/pull/8/commits/549a9732f002c780861d6fad5ec7e91ca0f0c60e)